### ISSUE TITLE
Concat strings?

### ISSUE THREAD

**Issue by salux** [DWilliames/google-sheets-sync-figma#14]
**Is your feature request related to a problem? Please describe.**
My developer counterparts sometimes concatenate strings from a string table into a single string output. I'd love to be able to add strings together (perhaps with some free text, too) for output into mocks.

**Describe the solution you'd like**
P1: Add strings together. Layer name example: #ID_FIRST_PART.1 + #ID_SECOND_PART
P2: Allow for free text, too. Layer name example: #ID_FIRST_PART.1 + " " + #ID_SECOND_PART

**Describe alternatives you've considered**
So far I've just left my mocks in free text so that they're not synced. Another alternative would be to create a second sheet in my doc and use rules to pull together strings there, but I haven't done that as it creates extra work for the developers to track.

**Additional context**
We're giving this tool a shot at helping us streamline some of our string tracking and localization efforts. I'm basically using this as a strict lookup table (hence the column # reference I tagged onto the examples above), and then once we're locked, I just copy/paste the name/value columns into a fresh doc to send to localization.

Thanks!

**Comment by prakashkarkee** [comment]
That is cool feature you mention with solution. Someone also issue #13  similar type of request.

**Comment by salux** [comment]
Yep, I think #13 is the same thing. Another Google Sheets sync plugin is available now, too, that lets you use multiple string variables in a single string render, but it's a little different than what I suggested above, so I'm posting it for reference.

This other plugin has you:
1) add multiple variable names to the layer name -> #FirstName#LastName
2) Write out the string in Figma, but mark parts to be replaced by the variables-> Hi {#FirstName} {#LastName}
